config.md

```diff
@@ -69,7 +69,7 @@ The runtime MUST mount entries in the listed order.
 ## Process configuration
 
 * **`terminal`** (bool, optional) specifies whether you want a terminal attached to that process. Defaults to false.
-* **`cwd`** (string, required) is the working directory that will be set for the executable.
+* **`cwd`** (string, required) is the working directory that will be set for the executable. This value MUST be an absolute path.
```

Comment:
I didn't view this as a misconfiguraton issue - per se.  I viewed @jonboulle's comment as:
1 - make sure we say that during a `start` the process must be run from the cwd (which I forgot to mention in my ops PR)
2 - make sure impls generate an error instead of creating the missing cwd - which some may think is a valid thing to do.  Both are valid options but we need consistency and in this case I think erroring out is the better choice.